src/_locales/de-DE.json

```diff
@@ -0,0 +1,94 @@
+{
+  "audio-device-communications": "System Default Communications",
+  "audio-device-default": "Standardeinstellung",
+  "audio-device-unknown": "Unbekanntes Gerät",
```

Comment:
Maybe make that:
    "audio-device-communications": "Standard Kommunikations Gerät"
    "audio-device-default": "Standard Gerät"
For consistency....